### PR TITLE
Gcw 2980 account details page not visible ie

### PR DESCRIPTION
**Pull request by akshaygwd**
### Ticket Link: 

https://jira.crossroads.org.hk/browse/GCW-2980

### What does this PR do?

Bug:
This PR fixes the issue of 
1. when new user login he is not redirected to the accout_detail page on ie.
2. On IE11, on the account details page for a new user, the email field contains '[object object]'

could you please review.

### CODE REVIEW

File: app/routes/post_login.js [hunk 1]
```diff
@@ -48,7 +44,7 @@ export default Route.extend({
         this.transitionTo("browse");
       }
     } else {
-      this.transitionTo("account_details");
+      window.location.replace("/account_details");
```
Comment (abhinav098) [inline app/routes/post_login.js]: @akshaygwd, please test it on all platforms. Transition is required. I would recommend to use `transitionTo` here.

File: app/routes/post_login.js [hunk 2]
```diff
@@ -15,10 +15,6 @@ export default Route.extend({
     this.set("isBookAppointment", params.queryParams.bookAppointment);
   },
 
-  model() {
-    return this.get("preloadService").preloadData();
-  },
```
Comment (abhinav098) [inline app/routes/post_login.js]: Why has this been removed ? `model` is required. right ? Have you tested this thoroughly on local before creating the PR ?

Comment (akshaygwd) [reply app/routes/post_login.js]: model is not used for post-login that's why I removed it

File: .circleci/config.yml [hunk 3]
```diff
@@ -355,28 +355,28 @@ workflows:
...
               branches:
-                only: /^(master|live)$/
+                only: /^(master|live|GCW-2980-account-details-page-not-visible-ie)$/
         - www-deploy:
             requires:
               - www_build
             filters:
               branches:
-                only: /^(master|live)$/
+                only: /^(master|live|GCW-2980-account-details-page-not-visible-ie)$/
```
Comment (namrataukirde) [inline .circleci/config.yml]: Make sure you will remove `GCW-2980-account-details-page-not-visible-ie` before merging PR.

**Pull request closed without merging**